app/celery/tasks.py

```diff
@@ -486,8 +488,21 @@ def persist_daily_sorted_letter_counts(day, file_name, sorted_letter_counts):
     dao_create_or_update_daily_sorted_letter(daily_letter_count)
 
 
+@dataclass
+class NotificationUpdate:
+    """
+    A NotificationUpdate is used to wrap a row of a DVLA response file.
+    `despatch_date` is optional to support files which contain it and those that don't
+    """
+
+    reference: str
+    status: str
+    page_count: str
+    cost_threshold: str
+    despatch_date: Optional[str] = ""
```

Comment:
I think this should probably be

```suggestion
    despatch_date: Optional[str] = None
```

Otherwise you're setting the default value to an empty string, which means that unless someone explicitly provides `None` it will always be a string, ie the `Optional` part is not necessarily accurate.

I appreciate it's probably a bit difficult to grok some of this at the moment as we don't have any actual static type-checking being done to provide feedback.

Reply:
Thanks, I've made the suggested change